importer/markdown/src/main/java/org/itsallcode/openfasttrace/importer/markdown/MdPattern.java

```diff
@@ -15,6 +15,8 @@ enum MdPattern
     // [impl->dsn~md.artifact-forwarding-notation~1]
 
     // @formatter:off
+    CODE_BEGIN("```\\w*\\s*"),
```

Comment:
Add a test with specified language, eg

q```Java

(Remove q, comment formatting breaks without)

Reply:
Done. Good reviewer. That was a test. :-)